test/test-0085.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+. ./test-common.sh
+
+# check state file locking
+cleanup 85
+
+preptest test.log 85 1
+
+touch state
+
+$RLR test-config.85 -v -f &
+
+sleep 1
```

Comment:
Waiting 1 second for logrotate to start in the background might not be sufficient in case the tests are runing in parallel on an overloaded build host.

Reply:
Increased the timeout values (waiting for the first run to acquire the lock and how long the first run will hold the lock)

Reply:
Fair enough.  We can implement explicit synchronization later if needed.